src/index.js

```diff
@@ -151,33 +125,30 @@ class TemplateContentScript extends ContentScript {
         qualificationLabel: 'energy_invoice'
       })
     ])
-    await this.clickAndWait('#header-deconnexion', '#idEspaceClientDiv')
   }
-
-  async tryAutoLogin(credentials) {
-    this.log('debug', 'Trying autologin')
-    const isSuccess = await this.autoLogin(credentials)
-    return isSuccess
+  async checkSession() {
+    this.log('debug', 'Starting checkSession')
+    await this.runInWorker('click', '#idEspaceClientDivMobile')
+    await this.waitForElementInWorker('#login-form')
+    // Here we wait for 3 secondes as the website could be a bit long to make the form interactive
+    // It may by present but not visible yet despite CSS is not actually hidding it
+    await sleep(3000)
```

Comment:
As i say in the comment, it is not possible to be 100% sure the form is active.
I'm first waiting for it, but once the `waitForElementInWorker` will return, the class in the CSS I use to detect the visibility will or will not be there (depending if a session is active). If it's hidden, then we'll never get it obviously, but if it's not, it will take some time to be fully interactive anyway, I couldn't find nothing changing in the page when the form appears (when available).